buildtools.bzl

```diff
@@ -0,0 +1,43 @@
+load("@bazel_skylib//lib:new_sets.bzl", "sets")
+
+_VALID_ASSET_NAMES = sets.make("buildifier", "buildozer")
+
+def _create_asset(name, platform, arch, version, sha256 = None):
```

Comment:
I think we might need to validate that these other args aren't `None`? I'm not sure if they will fail later or not

Reply:
Will do.

Reply:
Done.